py/desiutil/timer.py

```diff
@@ -149,6 +180,17 @@ def stopall(self):
             if 'stop' not in self.timers[name]:
                 self.stop(name)
 
+    def cancel(self, name):
+        """Cancel timer `name` and remove from timing log"""
+        t1 = time.time()
+        isotime = datetime.datetime.fromtimestamp(t1).isoformat()
+        if name in self.timers:
+            dt = t1 - self.timers[name]['start']
+            print(self._prefix('CANCEL'), f'Canceling timer {name} at {isotime} after {dt:.2f} seconds')
```

Comment:
Use `self._print()` instead?

Reply:
Good catch; I missed this one.  Updated to use `self._print`